### PR TITLE
Add new storage keys and storage names to store preferences and current sitter information.

### LINKED ISSUE #79: Add new storage names and keys for KeyValueStorage extensions to interact with Sitter and Preferences model
## Description
Add a new storage name and key for KeyValueStorage extensions to interact with the Sitter model and SelectedRole in Preferences storage

### Dev requirements and restrictions:
- add a new key to the extension `KeyValueStorage+Key`.
```swift
 static let currentSitter = "sitter"
 static let preferences = "preferences"
```
- add a new storage name  to the extension `KeyValueStorage+Name`.
```swift
static let currentSitter = "CurrentSitterStorage"
static let preferences = "PreferencesStorage"
```

### PR DESCRIPTION
**Pull request by NadzeyaShpakouskaya**
## Summary
This PR is a part of the work to resolve #79
Add new storage names and keys to store user preferences, and organize 2 flows (sitter and owner).

## Changes
- Add new keys to KeyValueStorage 
- Add new storage names to KeyValueStorage
- Add unit tests for them.

## Checks to complete
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] PR's title has a conventional commit style (short and descriptive).
- [x] Every non-private interface is documented properly.
- [x] I've added tests for my code (if it makes sense).
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

**Review by iBYarchitect — APPROVED** [review]
LGTM

**Review by julyakmita — APPROVED** [review]
LGTM.

Maybe let's do the PR's title more specific (because in future we will probably have PRs with the same title).
"Add new storage keys and storage name to store user preferences"

**Comment by NadzeyaShpakouskaya** [comment]
> LGTM.
> 
> Maybe let's do the PR's title more specific (because in future we will probably have PRs with the same title). "Add new storage keys and storage name to store user preferences"

Thanks for your point, I agree with you. Renamed the PR title.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Added 
 
 - Add `SitterMainTabView` for the sitter with stub view for tabs - [#108](https://github.com/ios-course/ironfoudation-team-project/pull/108)
+- Add new storage keys and names to store preferences and current sitter information  in `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
```
Comment (dersim-davaod) [inline CHANGELOG.md:12]: nit:
```suggestion
- Add new storage keys and names to store preferences and current sitter information in `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
```

Comment (dersim-davaod) [reply CHANGELOG.md:12]: nit:

In general, you can make it even simpler:
```suggestion
- Update storage keys and storage names to use with the `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
```

**Review by dersim-davaod — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (NadzeyaShpakouskaya) [reply CHANGELOG.md]: Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/109/commits/a0817ad7ed8484d31255be21fc31c16539bcb076

**Pull request merged**